### PR TITLE
2.6.0

### PR DESCRIPTION
**Pull request by alucas**

### CODE REVIEW

File: src/POGOProtos/Enums/BadgeType.proto [hunk 1]
```diff
@@ -40,4 +40,5 @@ enum BadgeType {
 	BADGE_TYPE_FAIRY = 35;
 	BADGE_SMALL_RATTATA = 36;
 	BADGE_PIKACHU = 37;
+	BADGE_UNOWN = 38;
```
Comment (jabbink) [inline src/POGOProtos/Enums/BadgeType.proto:43]: UN**KN**OWN or purposely abbreviated?

Comment (apavlinovic) [reply src/POGOProtos/Enums/BadgeType.proto:43]: Unown is correct

Comment (jabbink) [reply src/POGOProtos/Enums/BadgeType.proto:43]: Ok, sorry for wasting your time then :'(

**Comment by AeonLucid** [comment]
Looks good, thanks again! 👍

**Pull request merged**